### PR TITLE
Define specification sponsor and seed with spec approvers

### PR DESCRIPTION
**Pull request by jack-berg**
Originally proposed in https://github.com/open-telemetry/opentelemetry-specification/issues/3821, this PR formalizes the notion of a specification sponsor, which the TC and GC have been discussing for some time now.

A spec sponsor is a trusted collaborator of the TC. They are trusted to be assigned the type of complex specification issues that tend to require a fair bit of experience operating in the OpenTelemetry project. They can also sponsor issues such that other collaborators can drive the work under the help / guidance of the sponsor.

This idea should be thought of as an evolution and expansion of the role of a spec approver. It adds more privileges / responsibilities to that group and takes away the signal specific status. It allows us to grow the group of people who can contribute to the spec, and signal an additional level of trust in members. 

This PR seeds the spec sponsor pool with the existing spec approvers, but this is a good opportunity to check in with the current spec approvers to make sure they're aligned with the privileges / responsibilities and are still interested / able to contribute to the project.

### CODE REVIEW

**Comment by jack-berg** [comment]
cc @open-telemetry/technical-committee

File: community-membership.md
Comment (jpkrohling) [inline community-membership.md:315]:
```suggestion
complex and highly impactful to downstream language SIGs and to the
```

File: community-membership.md
Comment (jpkrohling) [inline community-membership.md:320]: Thinking about emerging areas (Profiling?) and areas where we don't have much representation (mobile/browser instrumentation), would it make sense to relax on this requirement on exceptional cases?

File: community-membership.md
Comment (jpkrohling) [inline community-membership.md:329]: How many hours per week are we talking about here? If we have an idea, it would be good to share here.

File: community-membership.md
Comment (jpkrohling) [inline community-membership.md:332]: The statements within parenthesis sound scary and discouraging. How about add a new session about turning emeritus? 

Like:

```markdown
### Emeritus sponsors

Sponsors who do not have the time commitment anymore to fulfill their responsibilities are expected to request to be moved to emeritus sponsors. If needed, the TC might also vote to move a sponsor to emeritus.
```

This should cover both the inactivity and the lack of confidence scenarios.

File: community-membership.md
Comment (jpkrohling) [inline community-membership.md:334]: > Sponsor issues

Is this a responsibility, or a privilege? Are sponsors _expected_ to sponsor issues within their areas of expertise, or are they allowed to be sponsors for issues in areas they care about?

File: community-membership.md
Comment (jpkrohling) [inline community-membership.md:329]: I find it difficult to find the privileges in this list. Here are the ones I gathered:

- approve issues / PRs
- sponsor issues (see comment below)

Is that motivation enough to become a spec sponsor?

Comment (jack-berg) [reply community-membership.md:329]: If we continue with the spec issue management workflow we've been experimenting with, every non-trivial issue requires sponsorship. The sponsor either does the work and makes the PR themselves, or works with an outside collaborator and sponsors them. Therefore, the ability to sponsor issues means the ability to move something through the spec. 

> Is that motivation enough to become a spec sponsor?

I lifted this presentation of "Responsibilities and privileges" from the other roles which are defined in this document. This argument can be applied to any other role. Why does anyone take on these otel roles which primarily come with more responsibility? Everyone has their own reasons but I suspect an overarching one is that named roles can help you get stuff done.

File: community-membership.md
Comment (jack-berg) [reply community-membership.md:334]: Privilege, mostly.

> Are sponsors expected to sponsor issues within their areas of expertise

I'm not sure. I lean towards yes, but its hard to compel people to do things in OSS. Is the TC expected to drive popular spec issues? I think they ought to because solving popular issues helps the project by helping users. But there's no enforcement mechanism for this. Similarly, I think sponsors ought to be cognizant of the popular issues for their areas of interest / expertise, but its hard to imagine compelling them to take responsibility for those.

> or are they allowed to be sponsors for issues in areas they care about

They should be allowed to sponsor any issue, but will naturally gravitate towards their areas of interest / expertise. Limiting to only their areas of interest / expertise would be too rigid.

File: community-membership.md
Comment (jack-berg) [reply community-membership.md:332]: I don't disagree, but for context, this language was lifted from the approvers responsibilities of this document: https://github.com/open-telemetry/community/blame/main/community-membership.md#L165

File: community-membership.md
Comment (jack-berg) [reply community-membership.md:329]: I don't have any idea. Same goes for maintainer, approver, triager, and TC membership. The time commitment varies by person. 

There might be a reasonable minimum. Like, its hard to imagine fulfilling the responsiveness responsibility without at least 1-2 hours per week. I would also expect that sponsors do their best to attend the specification SIG.

File: community-membership.md
Comment (jack-berg) [reply community-membership.md:320]: That's an interesting question. Profiling hasn't made it to the spec yet, but according to [the meeting notes](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit#heading=h.2oh8ox7yooz4) the group has been meeting since June 2022. Do we formally define contributor anywhere? Is a "contribution" limited to PRs, reviews of issues / PRs, or does participating in a SIG count?

Comment (jpkrohling) [reply community-membership.md:320]: @svrnm, we _just_ talked about this during today's triage session, and here's another situation where contribution isn't code. 

@jack-berg, we have this definition, which already supports non-code contributions, but @svrnm will make this more explicit in a future PR.

https://github.com/open-telemetry/community/blob/main/community-membership.md#member

File: community-membership.md
Comment (jpkrohling) [reply community-membership.md:329]: I acknowledge that the language is similar to the other roles, and I'm OK with having this change here look like those. My comments on this review were mostly taking the perspective of someone coming to the community and thinking why would they decide to be a spec sponsor in the first place. Perhaps framing it that way would open a door for changing the other descriptions to be more enticing to new collaborators as well.

Feel free to mark this and my other comments as resolved without changing the PR.

**Review by MrAlias — APPROVED** [review]

File: community-members.md
Comment (reyang) [inline community-members.md:58]: @jack-berg Now that we're consolidating the sponsors, do you want to revisit the list in this PR or a separate PR? For example, @iNikem used to be very active couple years ago but not anymore, I don't know if he is still willing to be listed here or moved to Emeritus.

**Review by reyang — APPROVED** [review]

File: community-members.md
Comment (jack-berg) [reply community-members.md:58]: I want to revisit in a separate PR. That way we can have a single group of emeritus sponsors, rather than trying to figure out how to represent emeritus for an approver concept which no longer exists.

File: community-membership.md
Comment (jack-berg) [reply community-membership.md:320]: Left a comment over on #2080. Do you think we need to do anything additional on this PR, or is it implied that the same types of contributions that count towards community membership count towards spec sponsor?

File: community-membership.md
Comment (jack-berg) [reply community-membership.md:332]: Updated to include. I think this is an improvement on the existing language. The existing sections can be updated to reflect this in a followup if needed.

File: community-membership.md
Comment (trask) [inline community-membership.md:340]: sponsors could be asked to step down if they aren't fulfilling any of their responsibilities, not sure this one needs to be called out specifically
```suggestion
- Demonstrate sound technical judgement
```

**Review by trask — APPROVED** [review]

File: community-membership.md
Comment (jpkrohling) [reply community-membership.md:320]: Nothing to change here, in my opinion.

**Review by jpkrohling — APPROVED** [review]

**Review by svrnm — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]
Looks good.
Also confirming that I am still interested in continuing. (As I was part of spec-metric-approvers, and hence part of the seed list for spec sponsor)

> This PR seeds the spec sponsor pool with the existing spec approvers, but this is a good opportunity to **check in with the current spec approvers to make sure they're aligned** with the privileges / responsibilities and are still interested / able to contribute to the project.

**Comment by pellared** [comment]
nit:

> It adds more [...] responsibilities to that group 

Shouldn't each "sponsor-to-be" acknowledge that they want to be a sponsor (e.g. by asking for an approval)?

**Review by djaglowski — APPROVED** [review]

**Review by djaglowski — APPROVED** [review]

File: community-members.md
Comment (arminru) [inline community-members.md:48]: These are only initially seeded with our current specs approvers but @open-telemetry/specs-approvers continues to exist independently, so we still need to document them somewhere. Do we want to list them here in this document, or in https://github.com/open-telemetry/opentelemetry-specification/blob/main/README.md, or https://github.com/open-telemetry/opentelemetry-specification/blob/main/CONTRIBUTING.md?

**Review by jsuereth — APPROVED** [review]

File: community-membership.md
Comment (arminru) [inline community-membership.md:340]:
```suggestion
- Frequently attend the specification SIG meeting where issues and PRs are synchronously discussed.

```
This might be implicit but I think it would be worth pointing this out as a responsibility.

**Review by jmacd — APPROVED** [review]
I think it makes sense to replace the current per-signal approver roles and add the new sponsorship role proposed here.

**Review by tedsuo — APPROVED** [review]

**Comment by trask** [comment]
> nit:
> 
> > It adds more [...] responsibilities to that group
> 
> Shouldn't each "sponsor-to-be" acknowledge that they want to be a sponsor (e.g. by asking for an approval)?

@kumoroku @Oberon00 @lzchen @iNikem - can you let us know if you want to be included in the new specification sponsor role or not?

@cijothomas @djaglowski @tedsuo @MrAlias - have approved already

@zenmoto - was moved to emeritus recently (#2117), so can probably be removed from specification sponsor list

File: community-membership.md
Comment (carlosalberto) [inline community-membership.md:357]: Do we go with super majority? i.e. 2/3? Not a blocking concern though.

**Review by pellared — APPROVED** [review]
I think that I could be added as a sponsor per https://github.com/open-telemetry/community/pull/2122

File: community-membership.md
Comment (jack-berg) [reply community-membership.md:359]: This language came from the requirements to become a maintainer:

> Unless stated otherwise in a SIG charter ratified by the Technical Committee,
a new maintainer is elected by vote of the existing maintainers of the SIG.
The vote is officially started when a pull request to add a new maintainer
is opened, and ends when the pull request is merged. The pull request may be
merged when the following conditions are met:
> - The person being nominated has accepted the nomination by approving the pull request
> - All maintainers have approved the pull request OR a majority of maintainers
  have approved the pull request and no maintainer has objected by requesting
  changes on the pull request. In the case that all maintainers have not given
  approval, the pull request should stay open for a minimum of 5 days before merging.

The super majority requirement is the same as the requirement to add members to TC:

> A candidate can be elected to the TC by the super-majority vote (greater than two thirds) of the existing TC members.

So there's a question of whether we want spec sponsor election to be more like maintainer or more like TC. The unanimous approval or abstain requirement to become a maintainer may help ensure the maintainers work better together since members are only added who all agree or are neutral on. The 2/3 supermajority requirement allows members to be elected which some people actively disapprove of.

Hard to know which voting requirement will be more appropriate for this function. We can always start with the unanimous approve or abstain requirement and later relax to be 2/3 supermajority.

Comment (djaglowski) [reply community-membership.md:359]: In my opinion the function of this group is more similar to that of the technical committee than any maintainer group. Reasonable and limited disagreement is important to ensure the specification evolves carefully in the right direction and the 2/3 supermajority rule best reflects this.

**Comment by Oberon00** [comment]
@trask Hi, tentatively, I'm fine with being moved to the new role so you can go ahead with this PR. I will need to figure out internally if I actually have the time to do any meaningful work for the OTel spec. It's possible that I ask for being moved to emeritus later.

File: community-members.md
Comment (jack-berg) [reply community-members.md]: Bad merge conflict resolution. Fixed!

File: community-members.md
Comment (jack-berg) [reply community-members.md:57]: Bad merge conflict resolution. Fixed! 

> Have we confirmed with these approvers that they no longer want to be sponsors?

Some have confirmed, we're waiting on confirmation from others: https://github.com/open-telemetry/community/pull/2070#issuecomment-2117947968

**Comment by iNikem** [comment]
> > nit:
> 
> > 
> 
> > > It adds more [...] responsibilities to that group
> 
> > 
> 
> > Shouldn't each "sponsor-to-be" acknowledge that they want to be a sponsor (e.g. by asking for an approval)?
> 
> 
> 
> @kumoroku @lzchen @iNikem - can you let us know if you want to be included in the new specification sponsor role or not? (update: I've also just pinged them on Slack)
> 

I am not currently active in OpenTelemetry project. At the present time it does not make sense to include me in spec sponsors.

File: community-members.md
Comment (pellared) [inline community-members.md:57]: Per https://github.com/open-telemetry/community/pull/2070#issuecomment-2122916461

```suggestion
```

**Comment by lzchen** [comment]
> @kumoroku @lzchen - can you let us know if you want to be included in the new specification sponsor role or not? (update: I've also just pinged them on Slack)

@trask 

Hey I am currently overloaded with other responsibilities and most likely will not have time to commit to be a sponsor but am definitely interested in the future if my situation changes! Also am willing to contribute more to specs to earn a sponsorship role in the future if need be.

File: community-membership.md
Comment (jack-berg) [reply community-membership.md:359]: I'm don't see any strong opinions from TC members so I'll resolve this comment for now but we should come back to it if we run into issues.

**Comment by jack-berg** [comment]
@open-telemetry/governance-committee I think this is ready to merge. Thanks!

File: community-members.md
Comment (djaglowski) [inline community-members.md:55]:
```suggestion
```

File: community-members.md
Comment (djaglowski) [inline community-members.md:63]:
```suggestion
- [Daniel Jaglowski](https://github.com/djaglowski), observIQ
- [David Poncelow](https://github.com/zenmoto), Splunk
```

**Review by djaglowski — COMMENTED** [review]
I've decided to focus on other areas and have moved myself to emeritus.

**Pull request merged**
